sklearn/linear_model/tests/test_ridge.py

```diff
@@ -715,3 +716,43 @@ def test_ridge_fit_intercept_sparse():
     assert_warns(UserWarning, sparse.fit, X_csr, y)
     assert_almost_equal(dense.intercept_, sparse.intercept_)
     assert_array_almost_equal(dense.coef_, sparse.coef_)
+
+
+def test_errors_and_values_helper():
+    r = _RidgeGCV()
```

Comment:
Could you give names that are more meaningful please (here and below in this function). It makes the code easier to read. Our usual rule is that we strive to avoid single-letter names (aside from the famous 'X', and 'y', which arguably are historical oversights).